docs/reference/intro.rst

```diff
@@ -15,14 +15,27 @@ compromising the quality of interactions and workflow.
 Configuration
 =============
 
-It's recommended you set up a configuration file through the interactive CLI utility.
+It's recommended you set up your D-Wave Cloud Client configuration through the
+:ref:`interactive CLI utility <interactiveCliConfiguration>`.
+
+D-Wave Cloud Client provides multiple options for configuring communication with
+a :term:`solver`:
+
+* One or more locally saved :ref:`configuration files <configurationFiles>`.
+* :ref:`Environment variables <environmentVariables>`
+* Direct setting of key values in functions
+
+These options can be flexibly used together.
+
+.. _configurationFiles:
 
 Configuration Files
 -------------------
 
-** THE FOLLOWING IS JUST DRAFT CONTENT **
-
-Candidates paths for configuration files are set by the D-Wave homebase_ package.
+If a D-Wave Cloud Client configuration file is not specified when instantiating a
+client or solver, auto-detection searches for candidate files in a number of standard
+directories, depending on your local system's operating system, you can list with the
+:func:`~dwave.cloud.config.get_configfile_paths` method.
 
```

Comment:
Maybe here is a good place to mention the `dwave config ls` command?
```
$ dwave config ls
/home/radomir/.config/dwave/dwave.conf

$ dwave config ls -m
/etc/xdg/xdg-ubuntu/dwave/dwave.conf
/usr/share/upstart/xdg/dwave/dwave.conf
/etc/xdg/dwave/dwave.conf
/home/radomir/.config/dwave/dwave.conf
./dwave.conf
```

Reply:
Done